docs/CHANGELOG.md

```diff
@@ -9,6 +9,124 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+#### New env vars
+
+`BLOCK_EMISSION_IDLE_WARNING_THRESHOLD` - controls global override for the time after which node will start logging warnings if no heads are received.
+`EVM_DISABLED` - will prevent any chains from being loaded at all, and any jobs that interact with the EVM cannot be created/loaded/run. Distinct from `ETHEREUM_DISABLED` which continue to work as before and will load chains, but will not connect to any nodes and still allows you to create jobs. This is a new recommended setting (replacing `ETH_DISABLED`) for jobs that don't need the EVM at all, e.g. Cron jobs on DYDX.
+`ETH_DEFAULT_BATCH_SIZE` - EvmDefaultBatchSize controls the number of receipts fetched in each request in the EvmConfirmer
```

Comment:
would a user know what the EvmConfirmer is?

Reply:
I dont even know what that is 😅